scipy/LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright (c) 2024, Dawit Gulta & Stephen Chen. All rights reserved.
```

Comment:
Why was this LICENSE file added? It's fine to add attribution at the top of a code file (memetic_climbing.py), inclusion of an extra license would require discussion, and for it to be compatible with scipy's.